Framework/Algorithms/inc/MantidAlgorithms/ReflectometryReductionOneAuto3.h

```diff
@@ -44,29 +44,22 @@ class DLLExport ReflectometryReductionOneAuto3
     std::string iVsLam;
   };
 
-  class RebinParams {
-  public:
+  struct RebinParams {
     RebinParams(const double qMin, const bool qMinIsDefault, const double qMax,
                 const bool qMaxIsDefault, const boost::optional<double> qStep)
         : m_qMin(qMin), m_qMinIsDefault(qMinIsDefault), m_qMax(qMax),
           m_qMaxIsDefault(qMaxIsDefault), m_qStep(qStep){};
 
-    double qMin() const { return m_qMin; };
-    bool qMinIsDefault() const { return m_qMinIsDefault; }
-    double qMax() const { return m_qMax; };
-    bool qMaxIsDefault() const { return m_qMaxIsDefault; }
-    double qStep() const { return *m_qStep; };
-    bool hasQStep() const { return m_qStep.is_initialized(); }
-    std::vector<double> asVector() const {
-      return std::vector<double>{qMin(), qStep(), qMax()};
-    };
-
-  private:
     double m_qMin;
```

Comment:
I think it's a more common convention to drop the `m_` prefix for public members - it makes it more readable when using them.